pkg/storkctl/migration.go

```diff
@@ -70,6 +72,7 @@ func newCreateMigrationCommand(cmdFactory Factory, ioStreams genericclioptions.I
 	createMigrationCommand.Flags().StringSliceVarP(&namespaceList, "namespaces", "", nil, "Comma separated list of namespaces to migrate")
 	createMigrationCommand.Flags().StringVarP(&clusterPair, "clusterPair", "c", "", "ClusterPair name for migration")
 	createMigrationCommand.Flags().BoolVarP(&includeResources, "includeResources", "r", true, "Include resources in the migration")
+	createMigrationCommand.Flags().BoolVarP(&includeVolumes, "includeVolumes", "r", true, "Include volumees in the migration")
 	createMigrationCommand.Flags().BoolVarP(&startApplications, "startApplications", "a", true, "Start applications on the destination cluster after migration")
```

Comment:
startApplications default value should be false here

Reply:
For manually triggered the default is true, for scheduled migrations it is false.